products/sle/main.pm

```diff
@@ -415,7 +415,7 @@ sub load_x11_remote {
     # load onetime vncsession testing
     if (check_var('REMOTE_DESKTOP_TYPE', 'one_time_vnc')) {
         loadtest 'x11/remote_desktop/onetime_vncsession_xvnc_tigervnc';
-        loadtest 'x11/remote_desktop/onetime_vncsession_xvnc_java';
+        loadtest 'x11/remote_desktop/onetime_vncsession_xvnc_java' unless sle_version_at_least('15');
```

Comment:
Please, since now on use is_sle for verions check: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/version_utils.pm#L146 so here it should be is_sle('<15'). sle_version_at_least is deprecated

Reply:
fixed

- desktopapps-remote-client1: http://10.67.17.30/tests/235
- desktopapps-remote-supportserver1: http://10.67.17.30/tests/234